src/base-testplane.ts

```diff
@@ -1,4 +1,4 @@
-import _ from "lodash";
+import _ from "./bundle/lodash";
```

Comment:
Will we have to always use import from "./bundle/lodash"?
Can it be automatically handled at compile-time so all "import _ from 'lodash'" would be replaced with "import _ from './bundle/lodash'"?